index.js

```diff
@@ -1,27 +1,171 @@
+
+
+
+
+  // Conexión de Mongoose a la base de datos:
 mongoose
-  .connect(MONGODB_URI, {
-    useCreateIndex: true,
-    useNewUrlParser: true,
-    useUnifiedTopology: true
-  })
+  .connect(dataConnectionString, { useCreateIndex: true, useNewUrlParser: true, useUnifiedTopology: true })
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
-    // Before adding any documents to the database, let's delete all previous entries
-    return self.connection.dropDatabase();
+    return self.connection.dropDatabase();  // Esto vacía lo que hubiera previamente en la base de datos antes de añadir nada
   })
+
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    Recipe.create({
+      title: 'Cardamom Coffe Buns',
+      level: 'Amateur Chef',
+      ingredients: ["3 cups flour",
+        "2 teaspoons fine sea salt",
+        "1 packet active dry yeast",
+        "½ cup whole milk",
+        "¼ cup water",
+        "8 tablespoons unsalted butter",
+        "1 egg, beaten",
+        "2 teaspoons ground cardamom",
+        "1 cup dark brown sugar"],
+      cuisine: 'American',
+      dishType: 'dessert',
+      image: "https://cravingsbychrissyteigen.com/wp-content/uploads/2019/11/101919_CRAVINGS_SET_2_CIARDAMOM_COFFEE_ROLLS-min-960x1200.jpg",
+      duration: 40,
+      creator: 'Chrissy Teigen'
+    })
+    
+      .then(newRecipe => {
+        console.log('Esta es la nueva receta:', newRecipe, 'la receta se llama:', newRecipe.title)
+        return Recipe.insertMany(data)
+      })
+
+      .then(elm => {
+        console.log('Estas son todas las recetas:', elm)
+        return Recipe.updateOne({ title: "Rigatoni alla Genovese" }, { duration: 100 }, { new: true })
+      })
+
+      .then(det => {
+        console.log('Resultado del cambio:', det, 'La recceta tiene una duración de:')
+        return Recipe.deleteOne({ title: "Carrot Cake" })
+      })
+
+      .then(info => {
+        console.log(info)
+        return mongoose.disconnect()
+      })
+
   })
+
   .catch(error => {
     console.error('Error connecting to the database', error);
   });
+
+
+
+
+// INTENTÉ HACER PROMESAS ENCADENADAS PRO PERO ME DA VARIOS ERRORES:
+
+  // .then(newRecipe => console.log('Esta es la nueva receta:', newRecipe, 'la receta se llama:', newRecipe.title))   // La nueva receta sale bien
+
+  // .then(Recipe.insertMany(data))
+
+  // .then(elm => console.log('Estas son todas las recetas:', elm))   // Esto me retorna 'undefined'
+
+  // .then(Recipe.updateOne({ title: "Rigatoni alla Genovese" }, { duration: 100 }, { new: true }))
+
+  // .then(det => console.log('Resultado del cambio:', det, 'La recceta tiene una duración de:'))   // Esto me retorna 'undefined'
+
+  // .then(Recipe.deleteOne({ title: "Carrot Cake" }))
+
+  // .then(info => console.log(info))   // Esto me retorna 'undefined'
+
+  // .then(mongoose.disconnect())   // Esto me rompe todo :)
```

Comment:
```suggestion
// INTENTÉ HACER PROMESAS ENCADENADAS PRO PERO ME DA VARIOS ERRORES:

  // .then(newRecipe => console.log('Esta es la nueva receta:', newRecipe, 'la receta se llama:', newRecipe.title))   // La nueva receta sale bien

  // .then(() => Recipe.insertMany(data))

  // .then(elm => console.log('Estas son todas las recetas:', elm))   // Esto me retorna 'undefined'

  // .then(() => Recipe.updateOne({ title: "Rigatoni alla Genovese" }, { duration: 100 }, { new: true }))

  // .then(det => console.log('Resultado del cambio:', det, 'La recceta tiene una duración de:'))   // Esto me retorna 'undefined'

  // .then(() => Recipe.deleteOne({ title: "Carrot Cake" }))

  // .then(info => console.log(info))   // Esto me retorna 'undefined'

  // .then(() => mongoose.disconnect())   // Esto me rompe todo :)
```
Recuerda que las promesas son funciones callback a las promesas devueltas.